src/options.c

```diff
@@ -320,6 +323,7 @@ void options_load(Options *options) {
     OPTION_INI_INT("lowmem", options->lowmem, 0, 1);
     OPTION_INI_INT("interlace", options->interlace, 0, 1);
     OPTION_INI_INT("flicker", options->interlace, 0, 1);
+    OPTION_INI_INT("ran_target_fps", options->ran_target_fps, 0, 65535);
```

Comment:
i think the maximum FPS should be 50 here, instead of 65535. any reason why you went this high (special case or something)?

Reply:
was just testing why the UI wasn't working. it's back to 50 now.